plugins/matomo-backend/config.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Config {
+  matomo: {
+    api_token: string;
+    api_url: string;
```

Comment:
Please respect the upstream documentation for plugin configuration and use `camelCase` here

Ref: https://backstage.io/docs/conf/defining/#guidelines

Reply:
Resolved